client/context.go

```diff
@@ -262,6 +263,13 @@ func (ctx Context) WithAux(isAux bool) Context {
 	return ctx
 }
 
+// WithLedgerHasProto returns the context with the provided boolean value, indicating
+// whether the target Ledger application can support Protobuf payloads.
+func (ctx Context) WithLedgerHasProtobuf(val bool) Context {
```

Comment:
@austinchandra Just to understand the context of this PR: Is Evmos creating a custom ledger app just for Evmos?

The SDK team is working on SIGN_MODE_TEXTUAL, which improves ledger signing in general, and  comes with a Cosmos-wide Ledger app. https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-050-sign-mode-textual.md

Reply:
Evmos plans to create a custom Ledger app in the future, which will probably attempt to integrate some variant of SIGN_MODE_TEXTUAL. For now, we are using the Ethereum app to sign Cosmos payloads using EIP-712 Typed Messages, which this PR helps to enable.